src/monodroid/CMakeLists.txt

```diff
@@ -116,6 +118,33 @@ set(TEST_COMPILER_ARGS_CPP ${XA_COMPILER_FLAGS}
   Wuninitialized
   )
 
+if(ENABLE_NDK)
+  if (ENABLE_CLANG_ASAN OR ENABLE_CLANG_UBSAN)
+	set(TEST_COMPILER_ARGS_CPP
+	  ${TEST_COMPILER_ARGS_CPP}
+	  fno-omit-frame-pointer
+	  fno-optimize-sibling-calls
+	  )
```

Comment:
I *think* this should align with the `s` in `set(`?

Reply:
No, that would be a wall of text - the entries are inside the list, they should be indented in relation to `set`